apps/fastlane/Fastfile

```diff
@@ -0,0 +1,48 @@
+# import Fastfile from the apple-code-signing repo to reuse signing lanes
+import_from_git(
+  url: "https://github.com/customerio/apple-code-signing.git",
+  branch: "main",
+  path: "fastlane/Fastfile"
+)
+# Use expand_path to import code so it automatically resolves the path based on the current file
+import File.expand_path('helpers/build_helper.rb', __dir__)
+import File.expand_path('helpers/version_helper.rb', __dir__)
+
+# Helper method to update pubspec version
+def update_pubspec_version(project_path, new_version)
+  pubspec_path = File.join(project_path, 'pubspec.yaml')
+
+  unless File.exist?(pubspec_path)
+    UI.user_error!("pubspec.yaml not found at #{pubspec_path}")
+  end
+
+  pubspec_content = File.read(pubspec_path)
+  new_pubspec_content = pubspec_content.gsub(/^version: .*/, "version: #{new_version}")
+
+  File.write(pubspec_path, new_pubspec_content)
+  UI.message("Updated version to #{new_version} in #{pubspec_path}")
+end
+
+# Lane to update Flutter app version
+lane :update_flutter_sdk_version do |options|
+  project_path = options[:project_path] || File.join(Dir.pwd, '../../..')
```

Comment:
Is `File.join(Dir.pwd, '../../..')` safe to use, or do you think it will result in bugs if someone moves a file and something breaks? 

I am OK with the code, but I wonder if maybe a `UI.message(` right above would help if it does break, we can fix it quickly. 

```suggestion
  project_path = options[:project_path] || File.join(Dir.pwd, '../../..')
  UI.message("updating version for files in path....")  
```

Reply:
Yup, it can surely break if the file is moved. To help with this, there is [already a message](https://github.com/customerio/customerio-flutter/pull/137/files#diff-d67e154c29ddfa99f01998d885b48404fa005f335d4efee735ef298efafa314dR31) just below this line:

```
UI.message("Updating sdk version to #{version_name} in #{project_path}")
```

Are you suggesting adding another message, or was it just missed?